rslib/src/storage/deck/all_decks_of_search_notes.sql

```diff
@@ -0,0 +1,9 @@
+SELECT nid,
+  did
```

Comment:
Would it make more sense to use the odid here? Currently, if you export notes in a filtered deck with deck names, the reimport will fail. (With a "deck missing" message; reporting that the specified deck is filtered instead would require some more work.)
I guess this usually doesn't matter because a deck column is only really useful if you want to add cards from outside Anki, but who knows.

Reply:
No strong preference on my end - it's a bit inconsistent with our apkg handling, but if it avoids a confusing error then perhaps it's the better choice.

Reply:
> it's a bit inconsistent with our apkg handling

Good point. Maybe we should leave it for now and come back to it after there's been some real-world usage?